scripts/build/docker-build-image

```diff
@@ -34,6 +39,11 @@ echo "  - container : $CONTAINER"
 echo "  - volumes   : " "${VOLUMES[@]}"
 echo "  - options   : " "${PASSTHROUGH[@]}"
 
+if [ -n "$PULL" ]; then
+    FROM_IMAGE=$(grep '^FROM ' "$DOCKERFILE" | head -1 | cut -d ' ' -f 2)
+    docker pull "$FROM_IMAGE"
+fi
+
```

Comment:
Hmm, why no just use the `--pull` option of docker build instead of doing possibly error-prone grepping?

Reply:
Because I did not know that such an option exists.

Reply:
But I'll update it using that...